gravitee-apim-e2e/lib/fixtures/management/ApisFaker.ts

```diff
@@ -14,21 +14,22 @@
  * limitations under the License.
  */
 import { ApiEntity, ApiEntityFlowModeEnum } from '@management-models/ApiEntity';
-import { MemberEntity } from '@management-models/MemberEntity';
 import { PageEntity } from '@management-models/PageEntity';
 import { Visibility } from '@management-models/Visibility';
 import { LoadBalancerTypeEnum } from '@management-models/LoadBalancer';
 import { Proxy } from '@management-models/Proxy';
 import { NewApiEntity } from '@management-models/NewApiEntity';
 import faker from '@faker-js/faker';
-import { UpdateApiEntity, UpdateApiEntityFlowModeEnum } from '@management-models/UpdateApiEntity';
 import { NewRatingEntity } from '@management-models/NewRatingEntity';
 import { RatingInput } from '@portal-models/RatingInput';
+import { Plan } from '@management-models/Plan';
 
 export interface ApiImportEntity extends ApiEntity {
-  members?: Array<MemberEntity>;
+  members?: Array<any>;
   pages?: Array<PageEntity>;
+  plans?: Array<Plan>;
   metadata?: any;
+  primaryOwner?: any;
```

Comment:
here maybe we could use  PrimaryOwnerEntity type

Reply:
:heavy_check_mark: